jobs/e2e_node/containerd/init.yaml

```diff
@@ -1,13 +1,25 @@
 #cloud-config
 
 runcmd:
-  - mount /tmp /tmp -o remount,exec,suid
+  - echo "This will configure built-in containerd for k8s tests. Containerd version is:"
```

Comment:
logs are showing up in `/var/log/cloud-init-output.log`, maybe we need to extend this to collect it into artifacts:

https://github.com/kubernetes/kubernetes/blob/a750d8054a6cb3167f495829ce3e77ab0ccca48e/test/e2e_node/services/logs.go#L73

Reply:
actually, on jounral based mchines it doesn't matter - `sudo journalctl -u cloud*` will have it covered